docs/reference/how-to/search-speed.asciidoc

```diff
@@ -326,3 +326,44 @@ queries, they should be mapped as a `keyword`.
 <<index-modules-index-sorting,Index sorting>> can be useful in order to make
 conjunctions faster at the cost of slightly slower indexing. Read more about it
 in the <<index-modules-index-sorting-conjunctions,index sorting documentation>>.
+
+[float]
+=== Use `preference` to optimize cache utilization
+
+There are multiple caches that can help with search performance, such as the
+filesystem cache, the <<shard-request-cache,request cache>> or the
```

Comment:
Maybe make filesystem cache a link to https://en.wikipedia.org/wiki/Page_cache ?